src/main/java/wolox/training/models/Book.java

```diff
@@ -17,9 +18,22 @@
     public Book() {
     }
 
+    public Book(String genre, String author, String image, String title, String subtitle,
```

Comment:
You don't need this new constructor. Users list should never be sent in the json body for create purposes, as it's always created empty.